static/includes/create-service-console.md

```diff
@@ -0,0 +1,52 @@
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and open your project.
+
+1. From your project, on the left-side menu, click <ConsoleLabel name="services"/> > **Create service**.
```

Comment:
nitpick: when you open your project, the **Services** page loads by default, no need to select it.